schemas/Clinical/index.js

```diff
@@ -178,6 +220,59 @@ const convertRegistrationDataToGql = data => {
   };
 };
 
+const convertClinicalSubmissionDataToGql = (data) => {
+  const submission = get(data, "submission", {});
+  const schemaErrors = get(data, "errors", {});
+  // convert clinical entities for gql
+  const clinicalEntities = [];
+  for (var clinicalType in submission.clinicalEntities) {
```

Comment:
don't use `var` here, introduces clinicalType to the global scope. use `let` instead.